### ISSUE TITLE
Please update Setting Up Workload Identity Federation in the GitHub Marketplace

### ISSUE THREAD

**Issue by alerickson0** [google-github-actions/auth#236]
### TL;DR

The docs for `Setting Up Workload Identity Federation` need to be updated. Because step 8 refers to the incorrect way of using the `--member` parameter.

### Expected behavior

For step 8) for the [Setting Up Workload Identity Federation](https://github.com/marketplace/actions/authenticate-to-google-cloud#setting-up-workload-identity-federation), the command should be:

`gcloud iam service-accounts add-iam-policy-binding "my-service-account@${PROJECT_ID}.iam.gserviceaccount.com" \
  --project="${PROJECT_ID}" \
  --role="roles/iam.workloadIdentityUser" \
  --member="principalSet://iam.googleapis.com/projects/${PROJECT_NUMBER}/locations/global/workloadIdentityPools/${WORKLOAD_IDENTITY_POOL_ID}/attribute.repository/${REPO}"`

Where the `--member` parameter should be of the format specificied [here](https://cloud.google.com/iam/docs/workload-identity-federation#impersonation).

### Observed behavior

The current step 8) for the [Setting Up Workload Identity Federation](https://github.com/marketplace/actions/authenticate-to-google-cloud#setting-up-workload-identity-federation), the command is:

`gcloud iam service-accounts add-iam-policy-binding "my-service-account@${PROJECT_ID}.iam.gserviceaccount.com" \
  --project="${PROJECT_ID}" \
  --role="roles/iam.workloadIdentityUser" \
  --member="principalSet://iam.googleapis.com/${WORKLOAD_IDENTITY_POOL_ID}/attribute.repository/${REPO}"`

### Action YAML

```yaml
N/A
```

### Log output

```text
And if you use the incorrect format for the `--member` parameter the error looks like this:

ERROR: (gcloud.iam.service-accounts.add-iam-policy-binding) INVALID_ARGUMENT: The member principalSet://iam.googleapis.com/*REDACTED*/attribute.repository/REPO_OWNER/REPO_NAME is of an unknown type. Please set a valid type prefix for the member.
```
